face/capture/src/main/java/com/simprints/face/capture/usecases/IsUsingAutoCaptureUseCase.kt

```diff
@@ -2,21 +2,20 @@ package com.simprints.face.capture.usecases
 
 import android.content.Context
 import androidx.preference.PreferenceManager
+import com.simprints.infra.config.store.models.ProjectConfiguration
 import com.simprints.infra.config.store.models.experimental
-import com.simprints.infra.config.sync.ConfigManager
 import dagger.hilt.android.qualifiers.ApplicationContext
 import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
 class IsUsingAutoCaptureUseCase @Inject constructor(
-    private val configManager: ConfigManager,
     @ApplicationContext private val context: Context,
 ) {
     private val preference = PreferenceManager.getDefaultSharedPreferences(context)
 
-    suspend operator fun invoke(): Boolean {
-        val isFeatureEnabled = configManager.getProjectConfiguration().experimental().faceAutoCaptureEnabled
+    operator fun invoke(projectConfiguration: ProjectConfiguration): Boolean {
+        val isFeatureEnabled = projectConfiguration.experimental().faceAutoCaptureEnabled
         val isOptionTurnedOnInSettings = preference.getBoolean(AUTO_CAPTURE_PREFERENCE_KEY, true)
         return isFeatureEnabled && isOptionTurnedOnInSettings
```

Comment:
Minor performance issue that's also present in the previous implementation - if `isOptionTurnedOnInSettings` is extracted as a function and placed inside the return statement, it will never be read when `isFeatureEnabled` is false (thus sparing shared preferences loading, reading & parsing).